tests/unit/test_charm.py

```diff
@@ -143,7 +146,7 @@ def test_active(
     harness.begin()
     harness.charm.on.config_changed.emit()
 
-    assert harness.model.unit.status == ActiveStatus()
+    assert harness.model.unit.status == ActiveStatus("")
```

Comment:
i noticed that in the integration tests we had this.  Do you think it's worth it to set the ActiveStatus("Ready") once it's reconciled?  I really don't know if it is or not

```
kubernetes-control-plane/0*  active    idle   0        10.246.153.231  6443/tcp    
kubernetes-worker/0*         active    idle   1        10.246.153.159  80,443/tcp  
```

Rather than 
```
kubernetes-control-plane/0*  active    idle   0        10.246.153.231  6443/tcp      Ready
kubernetes-worker/0*         active    idle   1        10.246.153.159  80,443/tcp    Ready
```